dnf/cli/commands/history.py

```diff
@@ -89,7 +125,7 @@ def configure(self):
                 logger.critical(self._require_one_transaction_id_msg)
                 raise dnf.cli.CliError(self._require_one_transaction_id_msg)
             demands.available_repos = True
-            commands._checkGPGKey(self.base, self.cli)
+            dnf.cli.commands._checkGPGKey(self.base, self.cli)
```

Comment:
Why you added `dnf.cli.`? All other commands uses `commands._checkGPGKey(self.base, self.cli)`.

Reply:
I was attempting to fix the lgtm-com warning you can see a bit above:
> 1 for Module is imported with 'import' and 'import from'

But the import just doesn't work, I couldn't figure out why. This change is a remnant of that.